app/build.gradle.kts

```diff
@@ -13,8 +13,10 @@ android {
     compileSdk = 31
     buildToolsVersion = "30.0.3"
 
+    namespace = "io.eugenethedev.taigamobile"
+
     defaultConfig {
-        applicationId = "io.eugenethedev.taigamobile"
+        applicationId = namespace!!
```

Comment:
- [ ]  Very strange and ugly move, IMHO. Why breed new entities? Maybe inline it back?

Reply:
That's just how it works now. `package` from the `AndroidManifest.xml` should be moved here as `namespace` parameter (AndroidStudio would show you warning if you have `package` in the manifest)